src/Xamarin.Legacy.Sdk/Sdk/Xamarin.Legacy.iOS.targets

```diff
@@ -44,8 +44,13 @@
       Condition=" '$(_FixupsNeeded)' == 'true' and $([MSBuild]::IsOSPlatform ('osx')) "
       AfterTargets="GetReferenceAssemblyPaths">
     <PropertyGroup>
-      <TargetFrameworkDirectory>$(FrameworkPathOverride)</TargetFrameworkDirectory>
+      <TargetFrameworkDirectory>$(FrameworkPathOverride);$(FrameworkPathOverride)Facades/</TargetFrameworkDirectory>
     </PropertyGroup>
+    <ItemGroup>
+      <DesignTimeFacadeDirectoryRoots Include="$(FrameworkPathOverride)" />
+      <DesignTimeFacadeDirectoryRoots Include="$(FrameworkPathOverride)Facades/" />
+      <DesignTimeFacadeDirectories Include="$(FrameworkPathOverride)Facades/" />
+    </ItemGroup>
```

Comment:
Should we do the same for Android and macOS, too?

Reply:
Very likely the answer is yes. I didn't test it though and it's almost midnight so it will have to wait till tomorrow... I have the test projects at hand, so it should be easy to verify.

Reply:
Could also update a project to reference Xamarin.Forms in here:

https://github.com/xamarin/Xamarin.Legacy.Sdk/tree/main/samples

Thanks!

Reply:
Android already seems to work correctly. macOS will need a similar fix as iOS.

Reply:
Done.